app/Models/Score/Best/Model.php

```diff
@@ -293,7 +293,7 @@ public function scopeWithType($query, $type, $options)
     {
         switch ($type) {
             case 'country':
-                $countryAcronym = $options['countryAcronym'] ?? $options['user']->country_acronym;
+                $countryAcronym = $options['countryAcronym'] ?? $options['user']->country_acronym ?? 'XX';
```

Comment:
I would think `$query->none()` would make more sense for non-existent users but I guess this works fine as long as there aren't any scores matching `XX` 🤔